extra-network/shadowsocks-rust/autobuild/ppc64el/prepare

```diff
@@ -0,0 +1,9 @@
+export SODIUM_USE_PKG_CONFIG=1
+
+abinfo "Setting HOME as /root to fix rustup ..."
+export HOME="/root"
+
+abinfo "Setting rust version as nightly to fix build ..."
+curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly -y
+export PATH=$HOME/.cargo/bin:$PATH
```

Comment:
Okay. You could have not used architectural sub-directories here (especially not by making a copy for each architecture - come on...!), I'd suggest that you use a big `if` branch instead.